x/ibc/testing/chain.go

```diff
@@ -369,8 +369,8 @@ func (chain *TestChain) GetPrefix() commitmenttypes.MerklePrefix {
 
 // NewClientID appends a new clientID string in the format:
 // ClientFor<counterparty-chain-id><index>
-func (chain *TestChain) NewClientID(counterpartyChainID string) string {
-	clientID := "client" + strconv.Itoa(len(chain.ClientIDs)) + "For" + counterpartyChainID
+func (chain *TestChain) NewClientID(clientType, counterpartyChainID string) string {
```

Comment:
```suggestion
func (chain *TestChain) NewClientID(clientType) string {
```

last argument isn't used anymore. we should remove it